scripts/deploy.js

```diff
@@ -0,0 +1,29 @@
+const hre = require("hardhat");
+
+const { deployBridge } = require("../lib/bridge");
+const { prepareToucanEnv } = require("../lib/toucan");
+
+async function deploy() {
```

Comment:
What is the difference between this script and https://github.com/regen-network/toucan-bridge/blob/b41af95ad0c1d93e561b2d907dae56f6961694dd/scripts/toucan-bridge.js? Seems like we can simply consolidate into one?

Reply:
This one deploys the toucan contracts as well as the bridge contract, then outputs a json blob with configuration data for the bridge service...